Decode hash to string, not byte sequence

stlk opened the pull request:
Not decoding caused comparison to fail on Python 3.6

gavinballard commented:
Thanks for the contribution @stlk! Could you please provide a test for this behaviour, so that I can make sure this change works OK on Python 2 and a range of Django versions?

stlk commented:
Thanks for writing this package and quick reply!

Method I changed is already tested. I managed to come up with small improvement. I changed the `WebhookTestCase.post_shopify_webhook` so the `HTTP_X_SHOPIFY_HMAC_SHA256` header uses `six.text_type` to be more version aware.

gavinballard commented:
Awesome, thanks @stlk!

gavinballard commented:
@stlk This change has been released to Pypi as v0.3.1. Thanks!
